### PR TITLE
Mepts 766

### PR DESCRIPTION
**Pull request by ningosi**
Match the new requirements for the CAT15 indicators

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeOnMohMqArtStartDateCalculation.java [hunk 1]
```diff
@@ -1,84 +0,0 @@
...
-import org.openmrs.module.eptsreports.reporting.calculation.AbstractPatientCalculation;
-import org.openmrs.module.eptsreports.reporting.calculation.BooleanResult;
-import org.openmrs.module.eptsreports.reporting.calculation.melhoriaQualidade.MohMqInitiatedArtCalculation;
-import org.openmrs.module.eptsreports.reporting.utils.EptsCalculationUtils;
-import org.openmrs.module.reporting.common.Birthdate;
-import org.openmrs.module.reporting.data.person.definition.BirthdateDataDefinition;
-import org.springframework.stereotype.Component;
-
-@Component
-public class AgeOnMohMqArtStartDateCalculation extends AbstractPatientCalculation {
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeOnMohMqArtStartDateCalculation.java:21]: @ningosi this is being used by quality improvements reports, did you see that?

**Review by psmatsinhe — COMMENTED** [review]
@ningosi please have a look on reference of the deleted files

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeOnMohMqArtStartDateCalculation.java (hunk 1 above)
Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeOnMohMqArtStartDateCalculation.java:21]: Yes, I saw that. I am the one who added that calculation that is no longer needed. I have cross referenced, build the code and tested.

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/calculation/generic/AgeOnMohMqArtStartDateCalculation.java:21]: We changed that to use end of the reporting period, which we don't need  calculation to accomplish that, hence be removed

**Review by psmatsinhe — APPROVED** [review]
Looks good

**Pull request merged**